batch/template2tsobject.ts

```
@@ -0,0 +1,30 @@
+import util from "util";
+
+
+const main = () => {
+  const promptTemplates = getAvailablePromptTemplates();
+  const promptData = util.inspect(promptTemplates, {
+    depth: null,
+    compact: false,
+    sorted: true,
+    breakLength: 120,
+  });
+  const promptTsExport = `export const data = ${promptData}\n`;
```

ESLint

[error] Remove this useless assignment to variable "promptTsExport".

(sonarjs/no-dead-store)